pkgs/tools/misc/tea/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchgit }:
+
+buildGoModule rec {
+  pname = "tea";
+  version = "v0.7.0";
+
+  src = fetchgit {
+    url = "https://gitea.com/gitea/tea";
+    rev = version;
+    sha256 = "sha256-Kq+A6YELfBJ04t7pPnX8Ulh4NSMFn3AHggplLD9J8MY=";
+  };
+
+  vendorSha256 = null;
+
+  doCheck = false;
```

Comment:
Please add a comment why you disabled tests.

Reply:
Apparently I didn't need to disable them. I removed it.